pkg/storage/cronjob_storage.go

```diff
@@ -0,0 +1,74 @@
+package storage
+
+import (
+	"context"
+
+	"jsin/database"
+	"jsin/logger"
+)
+
+type CronJob struct {
+	ID      int64
+	ChatID  string
+	CronJob string
+	Type    string
+}
+
+type CronJobStorage interface {
+	AddCronJob(ctx context.Context, cronJob string, jobType string) (int64, error)
+	ListCronJobDaily(ctx context.Context) ([]CronJob, error)
+}
+
+type CronJobStorageImpl struct {
+	gdb database.DBAdapter
+}
+
+var _ CronJobStorage = &CronJobStorageImpl{}
+
+func NewCronJobStorage() CronJobStorage {
+	gdb := database.GetDBInstance()
+	return &CronJobStorageImpl{
+		gdb: gdb,
+	}
+}
+
+func (i *CronJobStorageImpl) AddCronJob(
+	ctx context.Context,
+	cronJob string,
+	jobType string,
+) (int64, error) {
+	newJob := map[string]interface{}{
+		"chat_id":  ctx.Value("chatID"),
+		"cron_job": cronJob,
+		"type":     jobType,
+	}
+
+	result := i.gdb.DB().Table("cron_job").Create(newJob)
```

Comment:
you should create a model for this table, reference: 

https://github.com/hvn2706/jsin/blob/main/model/image.go
https://github.com/hvn2706/jsin/blob/main/pkg/storage/image_storage.go#L28-L48

then you can remove some magic keyword in the code